src/main/java/org/craftercms/studio/impl/v1/service/deployment/PublishingManagerImpl.java

```diff
@@ -407,11 +407,17 @@ public void processItem(CopyToEnvironment item) throws DeploymentException {
                     if (item.getOldPath().endsWith("/" + DmConstants.INDEX_FILE)) {
                         boolean hasRenamedChildren = false;
                         String fullPath = contentService.expandRelativeSitePath(item.getSite(), item.getOldPath().replace("/" + DmConstants.INDEX_FILE, ""));
-                        RepositoryItem[] children = contentRepository.getContentChildren(fullPath);
-                        if (children.length < 2) {
-                            deployer.deleteFile(item.getSite(), item.getOldPath().replace("/" + DmConstants.INDEX_FILE, ""));
-                        } else {
-                            hasRenamedChildren = true;
+                        if (contentService.contentExists(fullPath)) {
+                            try {
+                                RepositoryItem[] children = contentRepository.getContentChildren(fullPath);
+                                if (children.length < 2) {
+                                    deployer.deleteFile(item.getSite(), item.getOldPath().replace("/" + DmConstants.INDEX_FILE, ""));
+                                } else {
+                                    hasRenamedChildren = true;
+                                }
+                            } catch (Exception exc) {
+                                LOGGER.info("Error while checking children for moved content site " + item.getSite() + " old path " + item.getOldPath());
```

Comment:
Exception prints out info.  Should this be an Error instead?  I think so

Also, it swallows the error... so we need to at least print out the error.